src/transformers/models/lxmert/modeling_lxmert.py

```diff
@@ -705,6 +705,10 @@ def __init__(self, config, lxmert_model_embedding_weights):
         self.decoder.weight = lxmert_model_embedding_weights
         self.bias = nn.Parameter(torch.zeros(lxmert_model_embedding_weights.size(0)))
 
+    def maybe_resize_bias(self, new_size: int):
```

Comment:
Let's make this private

```suggestion
    def _maybe_resize_bias(self, new_size: int):
```

Reply:
Done.